simtools/simtel/simtel_histograms.py

```diff
@@ -38,51 +560,86 @@ def __init__(self, histogram_files, test=False):
         self._logger = logging.getLogger(__name__)
         if not isinstance(histogram_files, list):
             histogram_files = [histogram_files]
-        self._histogram_files = histogram_files
+        self.histogram_files = histogram_files
         self._is_test = test
-        self._list_of_histograms = None
         self._combined_hists = None
+        self._list_of_histograms = None
         self.__meta_dict = None
 
-    @property
-    def number_of_histograms(self):
-        """Returns number of histograms."""
-        return len(self.combined_hists)
-
-    def get_histogram_title(self, i_hist):
+    def calculate_trigger_rates(self, print_info=False):
         """
-        Returns the title of the histogram with index i_hist.
+        Calculate the triggered and simulated event rate considering the histograms in each file.
+        It returns also a list with the tables where the energy dependent trigger rate for each
+        file can be found.
 
         Parameters
         ----------
-        i_hist: int
-            Histogram index.
+        print_info: bool
+            if True, prints out the information about the histograms such as energy range, area,
+            etc.
 
         Returns
         -------
-        str
-            Histogram title.
+        sim_event_rates: list of astropy.Quantity[1/time]
+            The simulated event rates.
+        triggered_event_rates: list of astropy.Quantity[1/time]
+            The triggered event rates.
+        trigger_rate_in_tables: list of astropy.QTable
+            The energy dependent trigger rates.
         """
-        return self.combined_hists[i_hist]["title"]
+        triggered_event_rates = []
+        sim_event_rates = []
+        trigger_rate_in_tables = []
+        triggered_event_rate_uncertainties = []
+        for i_file, file in enumerate(self.histogram_files):
+            simtel_hist_instance = SimtelHistogram(file)
+            if print_info:
+                simtel_hist_instance.print_info()
+
+            logging.info(f"Histogram {i_file + 1}:")
+            logging.info(
+                "Total number of simulated events: "
+                f"{simtel_hist_instance.total_num_simulated_events} events"
+            )
+            logging.info(
+                "Total number of triggered events: "
+                f"{simtel_hist_instance.total_num_triggered_events} events"
+            )
 
-    @property
-    def list_of_histograms(self):
-        """
-        Returns a list with the histograms for each file.
+            obs_time = simtel_hist_instance.estimate_observation_time()
+            logging.info(
+                f"Estimated equivalent observation time corresponding to the number of"
```

Comment:
Add a space in to logging printout (it is now `number ofevents simulated`)

Reply:
Done.